vendor/github.com/dgrijalva/jwt-go/v4/errors.go

```diff
@@ -34,12 +34,12 @@ func wrapError(a, b error) error {
 type ErrorWrapper struct{ err error }
 
 // Unwrap implements xerrors.Wrapper
-func (w *ErrorWrapper) Unwrap() error {
+func (w ErrorWrapper) Unwrap() error {
```

Comment:
This reverts our fix from the other day.

Reply:
I fixed this in a new commit